docs/content/guides/rows/row-parent-child.md

```diff
@@ -2,7 +2,9 @@
 id: ivtc0o9b
 title: Row parent-child
 metaTitle: Row parent-child - JavaScript Data Grid | Handsontable
-description: Reflect the parent-child relationship of your data, using Handsontable's interactive UI elements such as expand and collapse buttons or an extended context menu.
+description:
+  Reflect the parent-child relationship of your data, using Handsontable's interactive UI elements such as expand and collapse buttons or an extended context
```

Comment:
Hello, I propose the following version for consideration:

"The feature reflects the parent-child relationship of the user's data by using Handsontable's interactive UI elements, such as **[Expand]** and **[Collapse]** buttons or available actions in the extended context menu."